ideas/096-message-tributes.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: 096-message-tributes
 title: Tribute to Talk
-status: Limbo
```

Comment:
Limbo is a keyword - maybe we can keep it in Limbo and leave a comment like you have below? Alt we could introduce a new keyword, but seems a bit messy

Reply:
Didn't realize—keep as Limbo!

Reply:
@rachelhamlin do you want to fix and we can merge this?

Reply:
@oskarth fixed :)